README.md

```diff
@@ -21,14 +21,15 @@ SwiftVideoBackground is an easy to use Swift framework that provides the ability
 
 ## Features
 
-- Play a video with one line of code
-- Play an array of videos sequentially
-- Automatically adjusts when device orientation changes
-- Automatically resumes video when app re-enters foreground
-- Loop videos *(optional)*
-- Mute sound *(optional)*
-- Darken videos so overlying UI stands out more *(optional)*
-- [Documentation](http://wilsonding.com/SwiftVideoBackground/)
+- [x] Play a video with one line of code
+- [x] Supports local videos `&&` videos from a web URL
+- [x] Automatically adjusts when device orientation changes
+- [x] Automatically resumes video when app re-enters foreground
+- [x] Pause, resume, restart, and other controls
+- [x] Loop videos *(optional)*
+- [x] Mute sound *(optional)*
+- [x] Darken videos so overlying UI stands out more *(optional)*
+- [x] [Documentation](http://wilsonding.com/SwiftVideoBackground/)
 
 ## Contents
```

Comment:
We should probably add entries/links to all of the subsections in Usage and the Issues section.

Reply:
✅